modules/Components/src/client/ACASThingBrowser.coffee

```diff
@@ -111,27 +86,92 @@ class ThingSimpleSearchController extends AbstractFormController
 					@$(".bv_thingSearchTerm").attr "disabled", false
 					@$(".bv_doSearch").attr "disabled", false
 
+	getQueryTerms: (queryTerms, searchTerm) ->
+		for queryValue in @configs
+			#Default is all display values are searchable
+			isSearchable = !queryValue.isSearchable? || queryValue.isSearchable != false
+			# Code Name is part of the default search so skip it here
+			if isSearchable && queryValue.key != "codeName"
```

Comment:
Don't quite follow why we need to filter out `codeName`, but if so, wouldn't we also want to filter out `recordedBy`?
Seems like a more generic check for whether a particular key is in the defaults would be preferable. I feel like I may be missing something here though.

Reply:
Your right about this.  Its just that the way I coded it, codeName is always searched on so there is nothing to do here (its already part of the queryParams).  RecordedBy is not always searched on (only if its in the display and isn't marked as isSearchable: false).  I'll clean this up to make it clear.  I realize now with the recordedBy  that there may be fields where you want to search on it, but not display it which I should as a feature.

Reply:
Retracting my last statement. Recorded By is always searched on so I just refactored the code to exclude those.  I could check the keys of queryTerms to check for their existence instead of the hardcoded way I wrote it.